extension/observer/ecsobserver/config.go

```diff
@@ -0,0 +1,105 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecsobserver
+
+import (
+	"os"
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
```

Comment:
config models has been moved around recently, need to rebase.

Reply:
Rebased and changed to `.../config`